src/app/components/Footer/__snapshots__/index.test.jsx.snap

```diff
@@ -48,6 +49,21 @@ exports[`Footer should render correctly 1`] = `
   margin: 0;
 }
 
+.c5 {
+  color: #FFFFFF;
+  font-weight: 700;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  display: block;
+  display: inline-block;
```

Comment:
display shouldn't be here twice

Reply:
It is an internal of styled-components' `extend` that this happened, only the latter would be applied though. However, this is a moot point. `extend` is [being deprecated](https://github.com/styled-components/styled-components/issues/1546) for `styled(Component)`, which provides a cleaner CSS and the same functionality for this use-case, we should use that.

Reply:
It is an internal of styled-components' extend that this happened, only the latter would be applied though. However, this is a moot point. extend is being deprecated for styled(Component), which provides a cleaner CSS and the same functionality for this use-case, we should use that.

Reply:
It is there twice because it is overwritten